### DISCUSSION TITLE
随便聊聊的语音服务的-语音识别识别设置无法正确获取到本地浏览器

### DISCUSSION THREAD

**Discussion by liuzj288** [lobehub/lobehub#2517] [General Question] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Official Preview

### 🌐 浏览器

Chrome

### 🐛 问题描述

随便聊聊这里是这样的
![image](https://github.com/lobehub/lobe-chat/assets/13833404/c7cfa9a3-72d0-414a-9458-165bbc2d9dae)

而agent下设置是这样的
![image](https://github.com/lobehub/lobe-chat/assets/13833404/ef48026c-7a74-465f-98d6-58ad1b468b50)

[a comment by an automated account is not shown]

**Comment by sxjeru** [comment, marked as answer]
一个是语音识别 in，一个是语音合成 out。

“随便聊聊”的语音合成选项在“默认助手”里。

[a comment by an automated account is not shown]
